src/icons/scripts/svgo-config.ts

```diff
@@ -0,0 +1,53 @@
+import { type Config } from 'svgo';
```

Comment:
```suggestion
import type { Config } from 'svgo';
```

Reply:
I'm not opposed to this change, as both of those statements would be dropped on the JS output. I primarily use named type imports for clarity, as they immediately identify types and accommodate future non-type imports without updating the statement.
Ex. 
```ts
import { type Config, loadConfig } from 'svgo';
```